services/catalog/service/migrations/committed/000008-add-tables-for-channels.sql

```diff
@@ -0,0 +1,72 @@
+--! Previous: sha1:ac06834674084dc237f4ac9168dfc87d7e85931c
+--! Hash: sha1:aa91e2e50535a2e9c6d7964bad84dc9af4299518
+--! Message: add-tables-for-channels
+
+DROP TABLE IF EXISTS app_public.channel CASCADE;
+CREATE TABLE app_public.channel(
+  id TEXT PRIMARY KEY,
+  title TEXT,
```

Comment:
That one should be possible to mark as "NOT NULL" as the channel service guarantees a non-empty title.